<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: rgb(56, 58, 89);
	color: rgb(235, 252, 255);
	border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(56, 58, 89);
	color: rgb(235, 252, 255);
	border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lbl_title_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>381</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 121, 199);
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Evaluate&lt;/span&gt;RDMs&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_sub">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>450</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 121, 199);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Step 3: Select ROIs&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_btn_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>230</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 121, 199);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ffffff;&quot;&gt;Selected ROIs:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>640</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(140, 164, 230);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>640</y>
       <width>201</width>
       <height>28</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(254, 121, 199);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Evaluate</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>440</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:outset;
background-color: rgb(140, 164, 230);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lst_selection">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>260</y>
       <width>201</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(140, 164, 230);
border-radius:10px;

</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lst_dataset">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>281</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(140, 164, 230);
border-radius:10px;
</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_dataset">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(254, 121, 199);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ffffff;&quot;&gt;Select ROIs:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <zorder>lbl_sub</zorder>
    <zorder>lbl_title_2</zorder>
    <zorder>lbl_btn_2</zorder>
    <zorder>btn_exit</zorder>
    <zorder>btn_start</zorder>
    <zorder>btn_add</zorder>
    <zorder>lst_selection</zorder>
    <zorder>lst_dataset</zorder>
    <zorder>lbl_dataset</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
